sktools/src/sktools/oldskfile.py

```diff
@@ -209,8 +220,21 @@ def equals(self, ref, atol=1e-10, rtol=1e-09):
                 warnings.warn('Mismatch in nuclear mass.')
                 return False
 
-        # Spline/polynomial repulsive comparison and
-        # Hyb/LC/CAM tag still missing in comparison
+        # Spline/polynomial repulsive is still missing in the comparison
+
+        if self.extratag and ref.extratag:
+            _, cam_params_self = parse_cam_from_content(
+                '\n'.join(self.extratag))
+            _, cam_params_ref = parse_cam_from_content(
+                '\n'.join(ref.extratag))
+            equal = np.allclose(
+                cam_params_self, cam_params_ref, rtol=rtol, atol=atol)
+        else:
+            equal = self.extratag == ref.extratag
+
+        if not equal:
+            warnings.warn('Mismatch in the Hybrids extra tag.')
```

Comment:
```suggestion
            warnings.warn('Mismatch in the hybrid functional extra tag.')
```
Any reason for Hybrid caps?

Reply:
Ah no, my original idea was to replace the `RangeSep` tag by a more general `Hybrid(s)` tag in the SK-files. But since this would break backwards compatibility I ultimately decided against it and forgot to revert the capitalization. I have applied your suggestion.